frontend/src/pages/SignIn/SignInContainer.tsx

```diff
@@ -16,11 +17,16 @@ const initSignInvalue: SignInFields = {
 };
 
 const SignInContainer: React.FC = () => {
+  const goToProfile = useNavigate();
+  const auth = useGetAuth();
+
   const formik = useFormik({
     initialValues: initSignInvalue,
     validationSchema: signInSchema,
     onSubmit: (values, { resetForm }): void => {
+      auth.mutateAsync(values);
       resetForm();
+      goToProfile(PATHS.profile);
```

Comment:
It's the wrong name for the function. In case it's going to the profile, why do we need to send the same route additionally
```suggestion
      useNavigate(PATHS.profile);
```

Reply:
Also we should redirect client only in case of success! Do we need to remove this call to the onSuccess?

Reply:
Wep, now `onSuccess + redirect`  implemented and working. Renamed const `goToProfile` to `navigateTo` , useNavigate cannt receive args